### PR TITLE
fix(trans): fix the msgstr for nn.flatten

### PR DESCRIPTION
**Pull request by 0xMashiro**
The original comment: 
**Reshapes the tensor by flattening the sub-tensor from dimension ``start_axis`` to dimension ``end_axis``.**

Fix its Chinese message string according to the example:

```
.. testcode::

        import numpy as np
        from megengine import tensor
        import megengine.functional as F

        inp_shape = (2, 2, 3, 3)
        inp = tensor(
            np.arange(36, dtype=np.int32).reshape(inp_shape),
        )
        oup = F.flatten(inp, 2)
        print(inp.numpy().shape)
        print(oup.numpy().shape)

    Outputs:

    .. testoutput::

        (2, 2, 3, 3)
        (2, 2, 9)
```

### CODE REVIEW

File: source/locale/zh_CN/LC_MESSAGES/api_zh/megengine.functional.po [hunk 1]
```diff
@@ -1243,15 +1243,15 @@ msgstr "单位矩阵"
 msgid ""
 "Reshapes the tensor by flattening the sub-tensor from dimension "
 "``start_axis`` to dimension ``end_axis``."
-msgstr "通过将子张量从 ``start_axis`` 维展平到 ``end_axis`` 维，实现对张量的重塑(reshape)。"
```
Comment (xxr3376) [inline source/locale/zh_CN/LC_MESSAGES/api_zh/megengine.functional.po:1246]: nice catch！

感觉这里不应该尝试逐词翻译，中英文的表达顺序逻辑是不同的。

感觉是不是这样翻译，更容易让母语是中文的人看得懂呢？

`将张量的 start_axis 维到 end_axis 维部分，展平为一维`

File: source/locale/zh_CN/LC_MESSAGES/api_zh/megengine.functional.po [hunk 2]
```diff
@@ -1243,15 +1243,15 @@ msgstr "单位矩阵"
...
 
 #: megengine.functional.nn.flatten:6 of
 msgid "The start dimension that the sub-tensor to be flattened. Default: 0"
-msgstr "子张量被展平时的初始维数。 默认： 0"
+msgstr "需要被展平的子张量的起始维数。 默认： 0"
 
 #: megengine.functional.nn.flatten:8 of
 msgid "The end dimension that the sub-tensor to be flattened. Default: -1"
-msgstr "子张量被展平时的最终维数。 默认： -1"
+msgstr "需要被展平的子张量的最终维数。 默认： -1"
```
Comment (xxr3376) [inline source/locale/zh_CN/LC_MESSAGES/api_zh/megengine.functional.po]: 最终 -> 终止？

**Review by xxr3376 — COMMENTED** [review]
Thanks for your contribution.

There are some mistakes in the original translation, seems that the guy who translates these part are not familiar with deep learning enough.

File: source/locale/zh_CN/LC_MESSAGES/api_zh/megengine.functional.po (hunk 1 above)
Comment (0xMashiro) [reply source/locale/zh_CN/LC_MESSAGES/api_zh/megengine.functional.po:1246]: 是的，这样更合理些~已经修改。

**Comment by xxr3376** [comment]
Please squash all commit into one, thx.

**Comment by xxr3376** [comment]
This PR is already merged, but due to the squash of copybara, GitHub shows a closed state instead.

**Pull request merged**